packages/remix-react/browser.tsx

```diff
@@ -24,7 +24,7 @@ import {
 declare global {
   var __remixContext: {
     url: string;
-    basename?: string;
+    basename: string;
```

Comment:
I think we can ensure this is always available from `ServerBuild` through `window.__remixContext` since we'll default it to `/`

Reply:
Scratch that - made it optional again in [d20b1b5](https://github.com/remix-run/remix/pull/8145/commits/d20b1b5149afe32aa9f9e4c1077a6bf3f7aee335) since non-vite builds won't have it